lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1082,16 +1088,11 @@ getNetworkInformation (_block0, bp, st) nl = do
         (bp ^. #getSlotLength)
         (bp ^. #getGenesisBlockDate)
 
-newtype ErrNextEpoch = ErrNextEpochCannotBeCalculated Iso8601Time
-
-calculateNextEpoch
-    :: W.SlotParameters -> UTCTime -> Either ErrNextEpoch ApiEpochInfo
-calculateNextEpoch sps time = case W.epochCeiling sps time of
-    Nothing -> Left $ ErrNextEpochCannotBeCalculated $ Iso8601Time time
-    Just en -> Right $ ApiEpochInfo
-        { epochNumber = ApiT en
-        , epochStartTime = W.epochStartTime sps en
-        }
+    -- Unsafe constructor for the next epoch. Chances to reach the last epoch
+    -- are quite unlikely in this context :)
+    unsafeEpochSucc :: HasCallStack => W.EpochNo -> W.EpochNo
+    unsafeEpochSucc = fromMaybe bomb . W.epochSucc
+      where bomb = error "reached final epoch of the Blockchain!?"
```

Comment:
**[silly or not question]**

even, when there is hard fork?

Reply:
Ho yeah. I mean, the only way we reach the last epoch is in a few thousand years from now. Capitalism would have ended humanity well before that.